src/libs/SidebarUtils.ts

```diff
@@ -648,7 +648,7 @@ function getOptionData({
     const lastActorDisplayName = getLastActorDisplayName(lastActorDetails);
     let lastMessageTextFromReport = lastMessageTextFromReportProp;
     if (!lastMessageTextFromReport) {
-        lastMessageTextFromReport = getLastMessageTextForReport(report, lastActorDetails, policy, reportNameValuePairs);
+        lastMessageTextFromReport = getLastMessageTextForReport(report, lastActorDetails, policy, !!reportNameValuePairs?.private_isArchived);
```

Comment:
Same here. Use `result.private_isArchived` and add `!!` to the definition above.